File: LENTABP10/src/main/java/com/lenta/lentabp10/MobileObjectsService/models/MB_S_31.java
package com.lenta.lentabp10.MobileObjectsService.models;

import com.fasterxml.jackson.annotation.JsonProperty;

import lombok.Getter;

// Модель справочника закрепления причин списания брака за секциями (WOB)
public class MB_S_31 {

    /// Тип задания на списание
    @JsonProperty("TASK_TYPE")
    @Getter private String taskType;

    /// Номер отдела
    @JsonProperty("SECTION_ID")
    @Getter private String sectionId;

    /// Причина движения
    @JsonProperty("REASON")
    @Getter private String reason;

    /// Текст к причине движения товара
    @JsonProperty("GRTXT")
    @Getter private String grtxt;

    public MB_S_31(String taskType, String sectionId, String reason, String grtxt) {
        this.taskType = taskType;
        this.sectionId = sectionId;
        this.reason = reason;
        this.grtxt = grtxt;
    }
}

File: writeoff_breakage/src/main/java/com/lenta/bp10/mobileObjectsService/models/MB_S_31.kt
package com.lenta.bp10.mobileObjectsService.models

import com.google.gson.annotations.SerializedName

// Модель справочника закрепления причин списания брака за секциями (WOB)
class MB_S_31(
        /// Тип задания на списание
        @SerializedName("TASK_TYPE")
        val taskType: String,

        /// Номер отдела
        @SerializedName("SECTION_ID")
        val sectionId: String,

        /// Причина движения
        @SerializedName("REASON")
        val reason: String,

        /// Текст к причине движения товара
        @SerializedName("GRTXT")
        val grtxt: String
)